test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts

```diff
@@ -136,8 +137,18 @@ describe("Proof of stake tests", () => {
         testUser1,
         await ProofOfStake.rewardPool(
           MGA_ASSET_ID,
-          newToken,
           newToken3,
+          newToken3,
+          Assets.DEFAULT_AMOUNT.muln(10e6),
```

Comment:
why this change? 
Before it was rewarding the pool of ( MGX , token ) with a token3, and now, it seems that it will be rewarding a pool of ( MGX , token3 ) with token3. that would mean that we are not strictly testing the same.  One is rewarding a pool with a token that does not belong to the pool, and the other is rewarding a pool with a token that belongs to it. 
where there any issue running the previous setup?

Reply:
It was problem on my local node. Change cancelled